### DISCUSSION TITLE
SageTV HD300 x2, 2x remotes, 2x IR extenders for sale

### DISCUSSION THREAD

**Discussion by blehnen** [OpenSageTV/Community#133] [Marketplace]
![IMG_3358](https://github.com/OpenSageTV/Community/assets/15485868/be209d4d-ed96-4dc4-8e79-0e74884957f6)
![IMG_3359](https://github.com/OpenSageTV/Community/assets/15485868/268a3b35-ead1-4b41-88fe-8649bd7a38d3)
![IMG_3360](https://github.com/OpenSageTV/Community/assets/15485868/3b45cc3f-9848-4b8b-91f4-47fa2ca113e0)

I was cleaning out my storage in the basement and found these.  Looking for $50 each + shipping, but would like to sell them all together.  Shipping would be flat rate USPS.  Medium box is $16, Large box is $21.85.  I'm not sure if I can fit all of this into a medium box, but I can check.  

They where working fine when turned off many years ago.  I'm a bit unsure of how to test them at this point, but I am happy to attempt it if the buyer can guide me a bit.

Hopefully someone can use these;  I moved to ShieldTV/Kodi many years ago.  I miss Sage, but we don't really watch 'TV' much anymore.

**Comment by TGsell1** [comment]
I’ll take 2 HD300s.  You can email me at ***@***.*** thanks Sent from my iPhoneOn May 20, 2024, at 11:04 PM, blehnen ***@***.***> wrote:﻿
IMG_3358.jpeg (view on web)
IMG_3359.jpeg (view on web)
IMG_3360.jpeg (view on web)
I was cleaning out my storage in the basement and found these.  Looking for $50 each + shipping, but would like to sell them all together.  Shipping would be flat rate USPS.  Medium box is $16, Large box is $21.85.  I'm not sure if I can fit all of this into a medium box, but I can check.
They where working fine when turned off many years ago.  I'm a bit unsure of how to test them at this point, but I am happy to attempt it if the buyer can guide me a bit.
Hopefully someone can use these;  I moved to ShieldTV/Kodi many years ago.  I miss Sage, but we don't really watch 'TV' much anymore.

**Reply by blehnen** [reply]
It seems that github blocks e-mail addresses from displaying - 

Can you write yours out in a notation such as - 

name site dot com 

I was able to get everything into a single medium rate flat box.

**Comment by TGsell1** [comment]
Tgsell    Optonline    NetThanks!Sent from my iPhoneOn May 21, 2024, at 1:40 PM, blehnen ***@***.***> wrote:﻿
It seems that github blocks e-mail addresses from displaying -
Can you write yours out in a notation such as -
name site dot com
I was able to get everything into a single medium rate flat box.

**Reply by blehnen** [reply]
Sold to [TGsell1](https://github.com/TGsell1) - thanks!
